packages/bpk-component-price/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bpk-component-price",
+  "version": "1.1.9",
```

Comment:
It should be `1.0.0`

Reply:
Or this'll  need to be `0.0.1` as our release process will version it to v1